config/airbrake.yml

```diff
@@ -1 +1,2 @@
-api_key: 'set yours here'
+project_key: 'set yours here'
```

Comment:
Changes required per https://github.com/airbrake/airbrake/blob/master/docs/Migration_guide_from_v4_to_v5.md. 
Should we put in a story to update the cookbooks?